### PR TITLE
Adds hook for headsup

### PR DESCRIPTION
**Pull request by lramos15**
This will post the chores to the whiteboard on a given cron schedule.

### CODE REVIEW

**Comment by zapdos26** [comment]
Is there a reason why you are adding it to this bot? 
Shouldn't the bot be independent of the whiteboard?

**Comment by lramos15** [comment]
> Is there a reason why you are adding it to this bot?
> Shouldn't whiteboard be independent of the bot?
> In other words, why are we going through chorebot as a middleman if we can pull directly from Google spreadsheets?


Because we use chorebot as a sort of service. Chorebot still functions without whiteboard and this just provides a webhook that fires whenever we want chorebot to push chores to the whiteboard. Chorebot is the centralized location for all things chores and should remain that way. This prevents us from duplicating any spreadsheet API code which is also goo for maintainability.

**Comment by zapdos26** [comment]
Then shouldn't you update the README accordingly then? Because the first line it states its function is a Slack integration, not a service to just process the chores.

File: .env.example [hunk 1]
```diff
@@ -4,6 +4,8 @@ SLACK_BOT_TOKEN=xoxb-YOURBOTTOKENHERE
 SLACK_SIGNING_SECRET=YOURSECRETHERE
 CHORES_CHANNEL=channel-here
 CRON_SCHEDULE=* * * * *
+WHITEBOARD_CRON_SCHEDULE=* * * * *
+WHITEBOARD_URL=whiteboard_url_here
```
Comment (ddbruce) [inline .env.example]: Please rename this to either `WHITEBOARD_SERVER_URL` or `HEADSUP_SERVER_URL`

**Comment by ddbruce** [comment]
Added env vars. Merging!

**Pull request merged**